pkg/asset/quota/openstack/openstack.go

```diff
@@ -7,10 +7,28 @@ import (
 	openstackprovider "sigs.k8s.io/cluster-api-provider-openstack/pkg/apis/openstackproviderconfig/v1alpha1"
 )
 
+const (
+	// These numbers should reflect what is documented here:
+	// https://github.com/openshift/installer/tree/master/docs/user/openstack
+	// https://github.com/openshift/installer/blob/master/docs/user/openstack/kuryr.md
+	// Number or ports here don't include the constraints needed for each machine, which are calculated later
+	portsConstraintKuryr                  = 15
```

Comment:
What is the difference between subnetsConstraintKuryr and subnetsConstraintWithKuryr? This is pretty confusing

Reply:
Can it be that `portsConstraintKuryr` is _without Kuryr_ while `portsConstraintWithKuryr` is _with_ Kuryr? :laughing:

Reply:
right, I was being asked to renamed it. Happy to rename it again if needed.

Reply:
sorry it was a typo, I'll fix it.